src/client/components/progress-bar/progress-bar.component.js

```diff
@@ -64,11 +64,13 @@ function ProgressBar({ startDate, currentDate, stages }) {
     const isPassed = day <= daysPassed;
     const restOfDays = day >= daysPassed;
     const classStyle = isPassed
```

Comment:
Can you change the name of classStyle also
Also, css props should be put in a css file